core/dnsserver/config_test.go

```diff
@@ -0,0 +1,122 @@
+package dnsserver
+
+import (
+	"github.com/coredns/caddy"
+	"testing"
+)
+
+func Test_keyForConfig(t *testing.T) {
+	type args struct {
+		blocIndex    int
+		blocKeyIndex int
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: "Test #1",
+			args: args{
+				blocIndex:    0,
+				blocKeyIndex: 0,
+			},
+			want: "0:0",
+		},
+		{
+			name: "Test #2",
+			args: args{
+				blocIndex:    0,
+				blocKeyIndex: 1,
+			},
+			want: "0:1",
+		},
+		{
+			name: "Test #3",
+			args: args{
+				blocIndex:    1,
+				blocKeyIndex: 0,
+			},
+			want: "1:0",
+		},
+		{
+			name: "Test #4",
+			args: args{
+				blocIndex:    1,
+				blocKeyIndex: 1,
+			},
+			want: "1:1",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := keyForConfig(tt.args.blocIndex, tt.args.blocKeyIndex); got != tt.want {
```

Comment:
Sorry, this function doesn't appear to be doing anything complicated - imo the disadvantages of extra boilerplate test code outweigh the possible benefits of test coverage (literally covering one line here).

Reply:
So do you want me to remove this test code? Let me know.

Reply:
I have a different opinion on this. The tests makes sure that the `fmt.Sprintf` is working as expected with a `:`. I agree we don't need many tests. But having 1 tests will ensure that things don't break. Thoughts?